Microsoft.Toolkit.Uwp.SampleApp/Shell.xaml.cs

```diff
@@ -201,6 +200,7 @@ public void RegisterNewCommand(string name, RoutedEventHandler action)
             };
 
             HideInfoArea();
+            NavigationFrame.Navigate(typeof(About));
```

Comment:
Why does this fix it?

Reply:
The error I was getting was very cryptic ("parameter is incorrect" in global exception handler) and took a lot of trial and error to narrow down where it was crashing. I believe (and don't know why on AU only), a resource is not ready earlier in the OnNavigated method when trying to navigate to a specific page and becomes available after the samples have loaded. Seems to work after testing few times in debug and release mode